extensions/net/ops_tls.rs

```diff
@@ -670,13 +672,17 @@ pub fn init<P: NetPermissions + 'static>() -> Vec<OpPair> {
   ]
 }
 
+#[serde_as]
 #[derive(Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct ConnectTlsArgs {
   transport: String,
   hostname: String,
   port: u16,
   cert_file: Option<String>,
+  #[serde_as(as = "Option<OneOrMany<_>>")]
+  #[serde(default)]
+  no_check_certificate: Option<Vec<String>>,
```

Comment:
I'm not so happy about adding additional dependency just to handle this situation. Are you sure it can't be handled using `serde` only? Maybe a custom deserializer would work here?

Reply:
I currently work on this.